content/ADR-145-pick-realm-v2.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: adr
+adr: 145
+title: Pick Realm Algorithm V2
+date: 2022-11-28
+status: Draft
+type: RFC
+spdx-license: CC0-1.0
+slug: /adr/TEMPLATE
+authors:
+  - agusaldasoro
+---
+
+## Abstract
+
+The logic on the realm scoring algorithm should be improved to take into consideration more variables in order for a server to be more eligible. As with the current model, any added variable needs to be modifiable through Feature Flags so that they can be changed in case of need without a release or a reboot of the server.
+
+## Context, Reach & Prioritization
+
+Some aspects that should be considered:
+
+- Resources: if a server is with the CPU at 90%, don't add more load to that server, same for memory
+- Catalyst version: if the version of the node is behind the network, it should be less eligible
+
+
+## Solution Space Exploration
+
+The current implementation as described in [Realm Picking Algorithm ADR](/adr/ADR-86) is expansible by adding new links. In this proposal two new links need to be added: resources and version, as the traffic will need to be prioritized to instances that have the latest version and more resources available.
```

Comment:
I cannot use the link `Realm Picking Algorithm ADR` (I tried to open it [here](https://github.com/decentraland/adr/blob/c6c8b45c01a67e3afb99ff216c55f8ca4c9ca2fa/content/ADR-145-pick-realm-v2.md)). Should we use an absolute link instead?

Reply:
@kevinszuchet This link works ok when you use the rendered version (you can get the URL from the PR's preview link: https://51d95b33.adr-cvq.pages.dev/adr/ADR-145).

Reply:
I copied the format from ADR-1